app/src/main/java/com/iyanuoluwa/imovie/data/remote/request/ApiInterfaces.kt

```diff
@@ -0,0 +1,47 @@
+package com.iyanuoluwa.imovie.data.remote.request
```

Comment:
Please take note of previously suggested changes

```suggestion
package com.iyanuoluwa.imovie.data.remote
```